pkg/model/components/clusterautoscaler.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package components
+
+import (
+	"k8s.io/kops/pkg/apis/kops"
+	"k8s.io/kops/pkg/apis/kops/util"
+	"k8s.io/kops/upup/pkg/fi"
+	"k8s.io/kops/upup/pkg/fi/loader"
+)
+
+// ClusterAutoscalerOptionsBuilder adds options for cluster autoscaler to the model
+type ClusterAutoscalerOptionsBuilder struct {
+	*OptionsContext
+}
+
+var _ loader.OptionsBuilder = &ClusterAutoscalerOptionsBuilder{}
+
+func (b *ClusterAutoscalerOptionsBuilder) BuildOptions(o interface{}) error {
+	clusterSpec := o.(*kops.ClusterSpec)
+	cas := clusterSpec.ClusterAutoscaler
+	if cas == nil || !fi.BoolValue(cas.Enabled) {
+		return nil
+	}
+
+	if cas.Image == nil {
+
+		image := "k8s.gcr.io/autoscaling/cluster-autoscaler:latest"
```

Comment:
I'm not a big fan of mutable tags such as "latest"; I'd prefer using "v1.19.0" (or whatever the highest known version is) for newer Kubernetes versions.

Reply:
I don't have any preference here.